src/routes/Calculator.jsx

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/extensions */
+import Calculator from '../components/Calculator.jsx';
+import styles from './styles/Calculator.module.css';
+
```

Comment:
- [ ] Dear @Mahamadou-Nouridine, You did great on the functionality, but to make it work I hade to edit some files. Please notice here you are importing a calculator component with a capital `C` (`Calculator.jsx`). but the file in your `components` directory is with a small `c` (`calculator.jsx`), this causes a problem when the app starts. Please take a look into it. 
![image](https://github.com/Mahamadou-Nouridine/math-magicians/assets/69077061/43528e80-af80-4d2e-acfc-292aff02f55a)